.github/workflows/ampli-merge-check.browser-v1.yml

```diff
@@ -1,10 +1,10 @@
-name: Ampli Merge Check (Browser)
+name: Ampli Merge Check (Browser V1)
 
 on:
   pull_request:
     paths:
-      - 'browser/javascript/**'
-      - 'browser/typescript/**'
+      - 'browser/javascript/v1'
```

Comment:
I think we still want the glob pattern to cover all subpaths in the examples e.g.`browser/javascript/v1/**`